nservicebus/upgrades/asp-saga-pruning.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Azure Storage Persistence upgrade Version 6.2.4 to 6.2.5
+summary: Instructions on how to patch Azure Storage Persistence when orphan saga index records appear.
+reviewed: 2016-06-23
+tags:
+ - upgrade
+related:
+ - nservicebus/sagas
+ - nservicebus/azure-storage-persistence
+---
+
+
+## Summary
+
+This document explains how to upgrade and patch a system for [Azure Storage Persistence bug #74](https://github.com/Particular/NServiceBus.Persistence.AzureStorage/issues/74) using NServiceBus.Azure hotfix release 6.2.5.
+
+This document explains how to upgrade and patch a system for [Azure Storage Persistence bug #26](https://github.com/Particular/NServiceBus.Persistence.AzureStorage/issues/26) using the NServiceBus.Azure hotfix release 6.2.4. 
+
+WARNING: When upgrading to NServiceBus.Persistence.AzureStorage Version 1 and above, the following upgrade will need to be performed prior to beginning any other upgrade steps.
```

Comment:
This exists in the 6.2.3 to 6.2.4 doc as well. The v6 to v1 upgrade points people to both of these docs and these warnings are intended to reinforce that people coming from the v6 to v1 doc can't skip these steps during the v6 to v1 upgrade